recipes/azure-sdk-for-cpp/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Usually we prefer the min cxx version to match the one declared in the recipe itself. Any specific reason this is not in this case?

Reply:
ah not at all nice catch i will, i based it off a recipe i made before and copied it blindly.